### DISCUSSION TITLE
Print Titles

### DISCUSSION THREAD

**Discussion by blogi-kriptovalyut** [mmistakes/minimal-mistakes#2653] [Q&A]
Hi, thanks for the topic,
I have installed jekyll-multiple-languages-plugin
on the page `about.md` I print the following variables:
![img-2020-08-13-23-10-57](https://user-images.githubusercontent.com/50487083/90182306-6cf6f180-ddba-11ea-8958-41eb3fa63dc6.png)

and in ru.yml file:

![img-2020-08-13-23-15-32](https://user-images.githubusercontent.com/50487083/90182622-ed1d5700-ddba-11ea-8aea-278435e3d7e4.png)

i disabled `jekyll seo tag` because it conflicts with `jekyll-multiple-languages-plugin`

i am printing the title of the page in the `head.html` file

`<title>{% t page.title %}</title>`

this line prints the title of the page `about.md` well (converts the variable `title: titles.pagination`) 
but! this does not print title from Front Matter `_posts` files

![img-2020-08-13-23-20-14](https://user-images.githubusercontent.com/50487083/90183244-d6c3cb00-ddbb-11ea-8581-d497e1bd3919.png)

![img-2020-08-13-23-22-34](https://user-images.githubusercontent.com/50487083/90183403-1be7fd00-ddbc-11ea-86a5-d7bdcd454610.png)

Question: how to make a header display condition?
If sometimes it is set for two different conditions?
1. in Front Matter `_posts` files
2. in the variable `title: titles.pagination` on the `about.md` page

Maybe there is a solution for this case? I want to use your theme
Thank.

**Comment by mmistakes** [comment]
Not sure I follow what you're asking.
This theme doesn't use `jekyll-seo-tag` plugin so you shouldn't need to disable it as it's not installed, unless you added it yourself.

I have no experience with the `jekyll-multiple-languages-plugin` plugin so can't really help there. Perhaps asking for help on Jekyll's forum might yield better results as it could be a configuration error on your part and not the theme's fault.

https://talk.jekyllrb.com/

**Comment by blogi-kriptovalyut** [comment]
> Не уверен, что понимаю, о чем вы спрашиваете.
> Эта тема не использует `jekyll-seo-tag`плагин, поэтому вам не нужно отключать его, поскольку он не установлен, если вы не добавили его самостоятельно.
> 
> У меня нет опыта работы с `jekyll-multiple-languages-plugin`плагином, поэтому я не могу здесь помочь. Возможно, обращение за помощью на форуме Jekyll может дать лучшие результаты, так как это может быть ошибка конфигурации с вашей стороны, а не ошибка темы.
> 
> https://talk.jekyllrb.com/

Sorry 
I meant the file `_includes/seo.html`
Not `jekyll-seo-tag`

**Comment by blogi-kriptovalyut** [comment]
Maybe this code will solve the problem of headers output
`<title>{% if page.title contains "pages." %}{% t page.title %}{% else %}{{ page.title }}{% endif %}</title>`

if the title is output in a variable `pages` in `ru.yml`,  then we display the title. 
If the title is specified as usual in `Front Matter`, then we display it
